src/pages/Patient/components/Patient__List.res

```diff
@@ -5,11 +5,31 @@ type ui =
   | Loaded
   | ShowPatient(PatientInfo.t)
 
+type covidStatus =
+  | POSITIVE
+  | SUSPECTED
+
 type state = {
   ui: ui,
   patients: array<PatientInfo.t>,
 }
 
+let getStatusType = (status: string) => {
+  switch status {
+  | "POSITIVE" => POSITIVE
+  | "SUSPECTED" => SUSPECTED
+  |_ => SUSPECTED
```

Comment:
why do we have "SUSPECTED" and _ and both pointing to the same variant `SUSPECTED`?

Reply:
I dont know how the data is entered. So to avoid NULL value I added this

Reply:
Okay... but if it's NULL should we put it as SUS? 😅 just curious